problemsets-mat223/linearalbegra.tex

```diff
@@ -5207,8 +5229,34 @@ \section*{Eigenvectors}
 	
 
 	\begin{parts}
-		\item Give an eigenvector for $T$.  What is the eigenvalue?
+		\item Give an eigenvector for $T$. What is the eigenvalue?
+			\begin{solution}
+				$\vec v=\mat{1\\1}$ is an eigenvector for $T$, with corresponding
+				eigenvalue $\frac{3}{2}$. 
+
+				We can see from the image that 
+				$T\mat{1\\1}=\mat{\frac{3}{2}\\\frac{3}{2}}=\frac{3}{2}\mat{1\\1}$.
+			\end{solution}
 		\item Can you give another?
+			\begin{solution}
+				Any scalar multiple of $\mat{1\\1}$ is also an eigenvector for $T$.
+
+				For any scalar $\alpha$, we have 
+				$T\left(\alpha\mat{1\\1}\right)=\alpha T\mat{1\\1}=\alpha\frac{3}{2}\mat{1\\1}$,
+				meaning $\alpha\mat{1\\1}$ is an eigenvector for $T$ with 
+				eigenvalue $\frac{3}{2}\alpha$.
+
+				More interestingly, $\mat{-1\\1}$ is an eigenvector with corresponding
```

Comment:
Move this sentence to be the second one. You don't need to prove anything. Just say, "Since T[-1 1] = 1/2*[-1 1]"

Reply:
I don't think I'm 100% clear on what you mean here, so let me know if what I did is what you were thinking.